src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -3170,7 +3168,12 @@ void SimZoneEquipment(EnergyPlusData &state, bool const FirstHVACIteration, bool
 
             ZoneEquipTypeNum = state.dataZoneEquipmentManager->PrioritySimOrder(EquipTypeNum).EquipTypeEnum;
 
-            ZoneCompNum = state.dataZoneEquip->ZoneEquipList(state.dataSize->CurZoneEqNum).EquipIndex(EquipPtr);
+            // TODO: need a way to set ParentIndex for other than UnitarySystem so ZoneCompNum is correct when using ParentIndex
+            if (ZoneEquipTypeNum == ZoneEquip::PkgTermACAirToAir || ZoneEquipTypeNum == ZoneEquip::PkgTermHPAirToAir) {
```

Comment:
I may have missed the WSHP here. ParentIndex was added to cover all 3 PTUnits. If there is only 1 equipment type in an input file or unit test, ParentIndex and EquipIndex will be the same so I can understand how I missed this. I did not attempt to fix how the component index is handeled by different equipmet types.

Reply:
That could very well be a merge mistake on my part. I'll check.